### PR TITLE
Add project shivammathur/setup-php

### PR DESCRIPTION
**Pull request by shivammathur**

### CODE REVIEW

File: projects/shivammathur/setup-php.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: setup-php
+repoUrl: https://github.com/shivammathur/setup-php
+description: GitHub action to set up PHP with extensions, php.ini configuration, coverage drivers, and various tools.
+languages:
+  - TypeScript
+  - JavaScript
+  - PHP
+  - PowerShell
```
Comment (ramonh) [inline projects/shivammathur/setup-php.mdx:9]:
```suggestion
  - Powershell
```

Fix minor Typo

File: projects/shivammathur/setup-php.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - Shell
+tags:
+  - PHP
+  - GitHub Actions
+  - Continuous integration
```
Comment (ramonh) [inline projects/shivammathur/setup-php.mdx:14]:
```suggestion
  - CI
```

Let's combine these into a shorter "CI" tag. We already list PHP as a language, so doing it as a tag is not necessary

**Review by ramonh — CHANGES_REQUESTED** [review]
Thanks so much for putting up your project! 

I made some comments below. Could you take a look please?

File: projects/shivammathur/setup-php.mdx (hunk 1 above)
Comment (shivammathur) [reply projects/shivammathur/setup-php.mdx]: Changed to Powershell

File: projects/shivammathur/setup-php.mdx (hunk 2 above)
Comment (shivammathur) [reply projects/shivammathur/setup-php.mdx]: Done

**Review by ramonh — APPROVED** [review]

**Comment by ramonh** [comment]
Thanks @shivammathur everything looks great!

Welcome to Port 🚢

**Pull request merged**